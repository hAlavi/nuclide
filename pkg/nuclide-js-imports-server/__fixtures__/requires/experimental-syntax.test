log(a?.b ?? 12_23);
log(<>Foo</>);
