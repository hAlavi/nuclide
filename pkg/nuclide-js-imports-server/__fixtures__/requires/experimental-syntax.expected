const React = require('React');

const a = require('a');
const log = require('log');

log(a?.b ?? 12_23);
log(<>Foo</>);
